ui/server/src/test/scala/pl/touk/nussknacker/ui/api/helpers/TestProcessUtil.scala

```diff
@@ -67,10 +68,20 @@ object TestProcessUtil {
       lastDeployedAction = lastAction.collect {
         case Deploy => createProcessAction(Deploy)
       },
-      json = json.map(_.copy(id = name, processingType = processingType)),
+      json = jsonData,
       history = history.getOrElse(Nil),
       modelVersion = None
     )
+  }
+
+  private def createEmptyJson(id: String, processingType: ProcessingType = Streaming) = {
+    val typeSpecificProperties = processingType match {
```

Comment:
And what about lite streaming data? Now we should always remember that after adding some new engine, we need to add here another impelmentation of data ? :(

Reply:
Yep, it's not very good.. But it's used only on tests, for creating sample scenario.

Reply:
I've added throwing excepion when type is not supported - now we know when someone try use not supported type.